### PR TITLE
SAT: script to create PR to migrate GA connector to `high` test strictness level.

### PR DESCRIPTION
**Pull request by alafanechere**
The `create_prs.py` script will create one PR per GA connectors to migrate to `high` test strictness level.

### What it does:
1. Iterate on all GA connectors in `source_definitions.yaml`
2. Create a branch for each GA connector
3. Locally migrate `acceptance_test_config.yml` to the latest format
4. Commit and push the changes on this branch
5. Open a PR for this branch
6. Run a SAT on this branch by posting a `/test` comment on the PR

**An example of the PR it creates can be found [here](https://github.com/airbytehq/airbyte/pull/19136)**

PR get created with the following labels:
* `area/connectors`
* `team/connectors-python` 
* `type/enhancement` 
* `test-strictness-level`

PR are added to the following project: `SAT-high-test-strictness-level`.

Please forgive me for duplicating a lot of code from `create_issues.py`. I think it's good enough in the context that this script will be used: a couple of times.

### CODE REVIEW

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,64 @@
+airbyte-cdk==0.7.0
```
Comment (evantahler) [inline airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/requirements.txt:1]: That's a big deps change - was this intentional?

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,64 @@
...
+python-dateutil==2.8.2
+pytzdata==2020.1
+PyYAML==5.4.1
+requests==2.28.1
+requests-cache==0.9.7
+requests-mock==1.9.3
+six==1.16.0
+smmap==5.0.0
+sortedcontainers==2.4.0
+-e git+ssh://git@github.com/airbytehq/airbyte.git@70679775b55c5bb1be7384114155924772885be0#egg=source_acceptance_test&subdirectory=airbyte-integrations/bases/source-acceptance-test
```
Comment (evantahler) [inline airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/requirements.txt:55]: This seems kind of scary

**Review by evantahler — CHANGES_REQUESTED** [review]
I think there might be something wrong with the dependencies (requirements.txt).  Other than that, 👍!

**Comment by evantahler** [comment]
Oh, and the tests should pass :D

**Comment by alafanechere** [comment]
> I think there might be something wrong with the dependencies (requirements.txt). Other than that, 👍!
@evantahler this is because I installed SAT package from the virtualenv I created for this tooling.
The `requirements.txt` in this PR is only scoped for the `tools/strictness_level_migration`. 
I just ran `pip install -e  .` from source-acceptance test. And then ran `pip freeze > requirements.txt` in `tools/strictness_level_migration`. This is not great but I don't think it's going to hurt. And SAT is not published to PyPi so we can't just `pip install sat`.

**Review by evantahler — APPROVED** [review]
Ok, that makes sense for the requirements!

**Pull request merged**